tests/MasterPeace/Bundle/UserBundle/Controller/UserBundleControllerTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace tests\MasterPeace\Bundle\UserBundle\Controller;
+
+use MasterPeace\Bundle\UserBundle\DataFixtures\ORM\LoadUserData;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\BrowserKit\Cookie;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+
+class UserBundleControllerTest extends WebTestCase
+{
+    public function testLogIn()
+    {
+        $client = self::createClient();
```

Comment:
Spėk :) bandžiau ir vakar. Nerandu kabliuko, kas ne taip. Jei copy/paste symfony dokumentacijos kodą - veikia, t.y. praeiną testą be problemų, nors tokio /admin route pas mus juk nėra..